### DISCUSSION TITLE
Birth Month and Counties: Maps of how birth month differs across the United States.

### DISCUSSION THREAD

**Discussion by astrowonk** [has2k1/plotnine#831] [2024 Plotnine Contest]
### Authors

Marcos Huerta

### Links

[Web](https://marcoshuerta.com), [Mastodon](https://vmst.io/@marcoshuerta), [Github](https://github.com/astrowonk)

### Full description

Plots and code are viewable on this Quarto-generated blog post - [Birth Month and Latitude](https://marcoshuerta.com/posts/birth-month-and-latitude/).

One of the parquet files of data is slightly too large for github, a zip file with the two parquet files needed for the plots is linked in the blog post above.

### Code repository

A standalone repository containing the quarto files (with python cells) is [here](https://github.com/astrowonk/plotnine_contest_2024).

**Comment by has2k1** [comment]
![fig-map-one-output-1](https://github.com/user-attachments/assets/1e4ff0f5-6a09-49f2-8631-6c8ebfb3be92)
<hr/>

![fig-scatter-one-output-1](https://github.com/user-attachments/assets/b5829318-c98b-4969-9f09-71c986af25b8)
<hr/>

![fig-map-two-output-1](https://github.com/user-attachments/assets/e4c249b4-bea1-47d1-9d40-19bde40fa611)

**Comment by EconomiCurtis** [comment]
Ahoy @astrowonk , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Reply by astrowonk** [reply]
Thanks! I filled out the form.

I  think I missed the winners announcement at Posit Conf; is that posted anywhere?

**Reply by has2k1** [reply]
It hasn't yet been posted.

**Comment by has2k1** [comment]
@astrowonk, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
